docs/adr/0002-use-timescale-for-metrics

```diff
@@ -0,0 +1,23 @@
+# 2. Use Timescale with PostgreSQL for Metrics
+
+Date: 2023-11-24
+
+## Status
+
+Accepted
+
+## Context
+
+We need to store certain metrics data, such as details about pull request and issues from GitHub and support requests from Slack. We're unable to get this information on demand using the APIs, because there's either no support with an existing Grafana plugin, it's not fast enough or it doesn't provide all the information we need.
```

Comment:
```suggestion
We need to store certain metrics data, such as details about pull requests and issues from GitHub and support requests from Slack. We're unable to get this information on demand using the APIs, because there's either no support with an existing Grafana plugin, it's not fast enough or it doesn't provide all the information we need.
```